src/Service/CaseEventHelper.php

```diff
@@ -70,30 +62,25 @@ public function createDocumentCaseEvent(CaseEntity $case, Party $sender, array $
             ->setSubject(CaseEvent::SUBJECT_HEARING_CONTRADICTIONS_BRIEFING)
             ->setReceivedAt($receivedAt)
             ->setCreatedBy($this->security->getUser())
+            ->setSenders([$sender->getName()])
         ;
 
         foreach ($documents as $document) {
             $caseEvent->addDocument($document);
         }
 
-        $this->createCaseEventRelations($caseEvent, CaseEventPartyRelation::TYPE_SENDER, [$sender]);
-
         $this->entityManager->persist($caseEvent);
         $this->entityManager->flush();
     }
 
-    private function createCaseEventRelations(CaseEvent $caseEvent, string $type, array $parties)
+    private function computeCaseEventSenderOrRecipient(array $parties, ?string $suffix): array
```

Comment:
`$suffix` should be renamed to `$additionalParties`, say, and it would make much more sense for it to be a list of names.